frontend/src/components/user/UserAvatar.vue

```diff
@@ -36,12 +43,21 @@ export default {
         ? userInitials(this.user)
         : campCollaborationInitials(this.campCollaboration)
     },
+    displayName() {
+      if (this.isLoading) {
+        return ''
+      }
+      return this.user
+        ? userDisplayName(this.user)
+        : campCollaborationDisplayName(this.campCollaboration)
+    },
     style() {
       return {
-        fontSize: this.size / 2.5 + 'px',
-        fontWeight: 400,
-        letterSpacing: '1px',
-        marginRight: '-1.5px',
+        color: contrastColor(this.color).toString({ format: 'hex' }),
```

Comment:
@carlobeltrame how would you do it here?

Reply:
`this.color` uses the color helpers internally. So `this.color` would return a hex color string according to my proposal. This means, you would simply be able to call `contrastColor(this.color)`.

I see that in this case, we convert the color object to hex string and then back to a color object. But I don't see this being a performance problem. I would prioritize code readability and developer experience over the very marginal performance benefit in this case.

Reply:
If there is ever a very expensive color calculation which we perform a lot and would require converting back and forth between hex strings and Color objects many times, I would simply write a dedicated helper function which performs this calculation. This helper could live in the common helpers, or in the specific component which uses it. Either way, 99.9% of the code won't have to know about `import Color from 'colorjs.io'`, `new Color(...)` and `.toString({ format: 'hex' })`.